sysdig/resource_sysdig_user_test.go

```diff
@@ -25,16 +27,16 @@ func TestAccUser(t *testing.T) {
 		},
 		Steps: []resource.TestStep{
 			{
-				Config: userWithSystemRole(rText()),
+				Config: userWithSystemRole(name),
 			},
 			{
-				Config: userWithName(rText()),
+				Config: userWithName(name),
 			},
 			{
-				Config: userWithoutSystemRole(rText()),
+				Config: userWithoutSystemRole(name),
 			},
 			{
```

Comment:
QQ Previously, each configuration had a different random name, but now all configurations take the same random name. Is this the intended behavior?

Reply:
I was debugging some tests and thought it was the issue. In the end issue was that the same email `terraform-test+user@sysdig.com` was used for all tests. So, since we now have secure tests running in parallel they try to create user with the same email and fail. This is some leftover and tbh it wont affect tests at all but I will revert those changes just to be sure.

Reply:
As I said previously `terraform-test+user@sysdig.com` was the same email used across all tests. So resource `sysdig_user.sample` cant change username (which is email) overtime. This is the reason why tests fails with infra drifts. Now I remember why I had this change 😄